components/shared/Field.tsx

```diff
@@ -1,21 +1,22 @@
-import React, { ReactNode } from 'react'
+import { ReactNode } from 'react'
 import { View } from 'react-native'
 
-import Typography from '@/components/shared/Typography'
+import Typography, { TypographyProps } from '@/components/shared/Typography'
 
 type Props = {
   label: string
   labelInsertArea?: ReactNode
   children: ReactNode
   errorMessage?: string
+  variant?: TypographyProps['variant']
 }
 // TODO associate control with label
 
-const Field = ({ label, children, labelInsertArea, errorMessage }: Props) => {
+const Field = ({ label, children, labelInsertArea, errorMessage, variant }: Props) => {
```

Comment:
Please set default variant here, not when used (line 19)

Reply:
oh yes a good catch